dockerfiles/theia/Dockerfile

```diff
@@ -168,6 +168,7 @@ COPY --from=build-result /che-theia-build/plugins /default-theia-plugins
     && find ${HOME} -exec sh -c "chgrp 0 {}; chmod g+rwX {}" \;
 
 COPY --chown=theia:root --from=build-result /che-theia-build /home/theia
+RUN rm -rf /home/theia/plugins
```

Comment:
should it be removed from build-result so we don't make grow this layer ?

Reply:
I agree. Updated.